### ISSUE TITLE
Decreased plasmalogens 

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9620]
**Preferred term label:**
Decreased circulating plasmalogen concentration

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
222765 
PMID: 20583171  1/1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Reduced plasmalogen concentration in the blood circulation.
Comment: Plasmalogens are a class of membrane glycerophospholipids that contain a vinyl-ether linked alkyl chain at the sn-1 position of the glycerol backbone and, typically, a polyunsaturated fatty acyl chain at the sn-2 position. 
PMID:30974122
parent -- Abnormal circulating phospholipid concentration

**Comment by pnrobinson** [comment]
added as HP_4000165
